### PR TITLE
'Tree' module replaced with TorusGeometry

### PR DESCRIPTION
**Pull request by Zlorak**
Due to some issues with loading local 3D models from the expo build, the Tree idea will not be integrated in the project until further research.

The tree .glb models are now replaced by TorusGeometry. Instanciated from inside R3F.

Also <Canvas /> had to be moved outside of the main styles.container since it doesn't like to be together with other components.

### CODE REVIEW

File: pomodoro-app/.expo/README.md
Comment (AdityaGudimella) [inline pomodoro-app/.expo/README.md:1]: It seems we shouldn't commit the `.expo` folder to git?

File: package-lock.json
Comment (AdityaGudimella) [inline package-lock.json:1]: Are the two package-lock.json files different?

**Review by AdityaGudimella — APPROVED** [review]
Looks good to me. Just had two questions.

**Review by GrantGrowcott — APPROVED** [review]
Cool Beans !!! I am one with the Torus

**Pull request merged**